Fix build with ports-gcc:
eg++: error: libgomp.spec: No such file or directory

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -596,7 +596,7 @@ include(SetCompilerLinkerFlags)
 
 if(CMAKE_C_COMPILER_ID STREQUAL Clang AND CMAKE_CXX_COMPILER_ID STREQUAL Clang AND NOT MSVC)
     # TODO: This should also be done for clang-cl.
-    include(LLVMToolchain)
+#    include(LLVMToolchain)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID STREQUAL Clang AND NOT MSVC)
@@ -723,7 +723,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID S
             endif()
         endif()
 
-        set(MY_C_FLAGS ${MY_C_FLAGS} -fopenmp)
     endif()
 
     if(CMAKE_CXX_COMPILER_ID STREQUAL GNU)
